modules/configuring-firewall.adoc

```diff
@@ -100,3 +100,13 @@ There are no special configuration considerations for services running on only c
 |`cloud.redhat.com/openshift`
 |Required for your cluster token.
 |===
++
+Operators require access to routes in order to perform health checks.
```

Comment:
s/routes in order to/routes to/ ?
We had a discussion about this in [team-docs Slack](https://coreos.slack.com/archives/C85JYPHL3/p1596640807367700) recently. ISG says to remove it "in most cases" but I actually see a case for leaving it here because it gives a little breathing room between "access to" and "routes to". ❔ 

You could also try rewording to something like, "Operators require route access to perform health checks" or "Operators require access to routes for performing health checks". Or just leave it as it is! ¯\_(ツ)_/¯